integration/airflow/tests/extractors/test_python_extractor.py

```diff
@@ -0,0 +1,39 @@
+import inspect
+import os
+from unittest.mock import patch
+
+from airflow.operators.python_operator import PythonOperator
+
+from openlineage.airflow.extractors.example_dag import python_task
```

Comment:
it would be helpful to rename this task function to something informative - e.g., `python_task_getcwd`

Reply:
Done.